client/clientd/src/bin/cli.rs

```diff
@@ -0,0 +1,49 @@
+use anyhow::Result;
+use clap::{Parser, Subcommand};
+use clientd::call;
+use minimint_api::module::__reexports::serde_json;
+use serde::Serialize;
+
+#[derive(Parser)]
+#[clap(author, version, about = "a json-rpc cli application")]
+struct Cli {
+    /// print unformatted json
+    #[clap(takes_value = false, long = "raw", short = 'r')]
+    raw_json: bool,
+    /// call JSON-2.0 RPC method
+    #[clap(subcommand)]
+    command: Commands,
+}
+#[derive(Subcommand)]
+enum Commands {
+    /// rpc-method: info()
+    Info,
+    /// rpc-method: pending()
+    Pending,
+}
+#[tokio::main]
+async fn main() {
+    let args = Cli::parse();
+
+    match args.command {
+        Commands::Info => {
+            print_json(call("", "/getInfo").await, args.raw_json);
+        }
+        Commands::Pending => {
+            print_json(call("", "/getPending").await, args.raw_json);
```

Comment:
It's kind of weird how these are camel case but `FederationApi` and gateway API are snake case. I don't have a preference between them, but I think we should probably be consistent?